x-pack/plugins/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/index.test.tsx

```diff
@@ -66,4 +70,20 @@ describe('EditOutputFlyout', () => {
     expect(utils.queryByLabelText('Client SSL certificate')).not.toBeNull();
     expect(utils.queryByLabelText('Server SSL certificate authorities')).not.toBeNull();
   });
+
+  it('should show a callout if encrypted key are not set and the flyout if the output provided is a logstash output', async () => {
```

Comment:
I realize this is just for testing, but it's a bit garbled. Do you mean to say something like: "should show a callout in the flyout if the selected output is logstash and no encrypted key is set"